app/controllers/users.py

```diff
@@ -1464,5 +1464,158 @@ def post(self):
             status='success',
             message=f'Successfully sent {emails_sent} reminder emails',
             total_users_processed=len(inactive_users),
-            errors=errors if errors else None
-        ), 201
+            errors=errors if errors else None)
+
+
+class GoogleOAuthView(Resource):
+    def get(self):
```

Comment:
It would be better if these both had individual functions under this class rather than sharing handle_oauth_request